{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:community:*:*:*",
            "versionEndExcluding": "11.11.7",
            "versionStartIncluding": "11.11.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*",
            "versionEndExcluding": "11.11.7",
            "versionStartIncluding": "11.11.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:community:*:*:*",
            "versionEndExcluding": "12.0.4",
            "versionStartIncluding": "12.0.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*",
            "versionEndExcluding": "12.0.4",
            "versionStartIncluding": "12.0.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:community:*:*:*",
            "versionEndExcluding": "12.1.2",
            "versionStartIncluding": "12.1.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*",
            "versionEndExcluding": "12.1.2",
            "versionStartIncluding": "12.1.0",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2019-5461"
    },
    "affects": {
      "vendor": {
        "vendor_data": []
      }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "An input validation problem was discovered in the GitHub service integration which could result in an attacker being able to make arbitrary POST requests in a GitLab instance's internal network. This vulnerability was addressed in 12.1.2, 12.0.4, and 11.11.6."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-20"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://about.gitlab.com/2019/07/29/security-release-gitlab-12-dot-1-dot-2-released/",
          "refsource": "MISC",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://about.gitlab.com/2019/07/29/security-release-gitlab-12-dot-1-dot-2-released/"
        },
        {
          "name": "https://gitlab.com//gitlab-org/gitlab-ce/issues/54649",
          "refsource": "CONFIRM",
          "tags": [
            "Exploit",
            "Vendor Advisory"
          ],
          "url": "https://gitlab.com//gitlab-org/gitlab-ce/issues/54649"
        },
        {
          "name": "https://hackerone.com/reports/446593",
          "refsource": "MISC",
          "tags": [
            "Third Party Advisory"
          ],
          "url": "https://hackerone.com/reports/446593"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV2": {
      "acInsufInfo": false,
      "cvssV2": {
        "accessComplexity": "LOW",
        "accessVector": "NETWORK",
        "authentication": "SINGLE",
        "availabilityImpact": "NONE",
        "baseScore": 4,
        "confidentialityImpact": "NONE",
        "integrityImpact": "PARTIAL",
        "vectorString": "AV:N/AC:L/Au:S/C:N/I:P/A:N",
        "version": "2.0"
      },
      "exploitabilityScore": 8,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainOtherPrivilege": false,
      "obtainUserPrivilege": false,
      "severity": "MEDIUM",
      "userInteractionRequired": false
    }
  },
  "lastModifiedDate": "2019-09-10T18:27Z",
  "publishedDate": "2019-09-09T17:15Z"
}